R-packages/evalcast/R/manage_cards.R

```diff
@@ -91,3 +91,34 @@ intersect_averagers <- function(cards, grp_vars, avg_vars) {
   }
   cards %>% right_join(averagers_intersected)
 }
+
+#' Scale error measures based on those of a particular forecaster.
+#'
+#' @param score_card score_card like that returned by `evaluate_predictions()`
+#' @param vars vector of column names in `score_card` to normalize
+#' @param denom_forecaster name of forecaster in `score_card$forecaster` column by whose error
+#'   values the remaining forecasters' errors will be scaled
+#' @param err_cols vector of column names in `score_card` that contain error measures.
+#'   Elements of `err_cols` that are not in `vars` will be dropped from the final output to
+#'   avoid a mix of scaled and unscaled measures.
+#'
+#' @return
+#' @export
+scale_by_forecaster <- function(score_card, vars, denom_forecaster, 
+                                err_cols = c("ae", "wis")) {
+  df_list <- map(vars, function(var) {
```

Comment:
Can you add a few checks here to ensure that the necessary stuff is in the `score_card` and produce appropriate warnings/errors? For example, make sure `denom_forecaster` is available, and that there are others to scale. Check that the card has a "forecaster" column and contains (I guess at least 1) of the `err_cols`?

Reply:
Done.